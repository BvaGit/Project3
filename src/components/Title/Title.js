import React, { Component } from 'react';
import './Title.scss';

class Title extends Component {
    render () {
        const { textBig, textSmall} = this.props;
        return (
            <div className="title">
                <h1 className="title__big">{textBig}</h1>
                <br/>
                <h2 className="title__small">{textSmall}</h2>
            </div>
        )
    }
}

export default Title;
