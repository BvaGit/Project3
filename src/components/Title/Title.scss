@import '../../utils/variables.scss';

.title {
    text-align: left;
    margin: 0 auto;
    width: 50%;
    color: $title-color;
    letter-spacing: 1.5px;
    cursor: default;
    &__big {
        font-size: 40px;
        color: $l-body-text-color;
    }
    &__small {
        font-size: 30px;
        color: $l-body-text-color;
    }
}
