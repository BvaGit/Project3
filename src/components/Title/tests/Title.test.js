import React from 'react';
import { shallow, mount } from 'enzyme';
import Title from '../Title';


describe('Title', () => {
    it("Should match snapshot", () => {
        const component = shallow(<Title />)
        expect(component.html()).toMatchSnapshot()
    })
})