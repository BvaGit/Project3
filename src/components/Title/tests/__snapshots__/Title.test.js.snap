// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title Should match snapshot 1`] = `"<div class=\\"title\\"><h1 class=\\"title__big\\"></h1><br/><h2 class=\\"title__small\\"></h2></div>"`;
